astro/cli/astro/cmd/cmds.go

```diff
@@ -113,6 +113,16 @@ var planCmd = &cobra.Command{
 	},
 }
 
+var versionCmd = &cobra.Command{
+	Use:                   "version",
+	DisableFlagsInUseLine: true,
+	Short:                 "Print astro version",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		fmt.Printf("astro version %s (%s) built %s\n", version, commit, date)
```

Comment:
When I run version built locally, I'm getting 
```
astro version DEV (none) built unknown
```
Can we change this message so that it's clear what none and unknown stand for?
If it's hard to fill out the git hash and build time, you can skip this information all together or add tags, so that it'd be
```
astro version DEV (hash: unknown) built time: unknown
```

Reply:
Sure, I think I'll just remove the hash and build time altogether.

Reply:
OK, I updated this to address your comments.

Git hash and current date are now determined within Go at build time. This means the hash and date will be available even when compiling locally, e.g.:

```
astro version dev (8085049) built 2018-12-28T14:20:48+01:00
```

When the hash can't be determined:

```
astro version dev (hash unknown) built 2018-12-28T14:21:14+01:00
```

The releaser can update the version using linker flags:

```
dan-mbp-work:astro dan$ go build -o bin/astro -ldflags="-X github.com/uber/astro/astro/cli/astro/cmd.version=1.2.3" github.com/uber/astro/astro/cli/astro
dan-mbp-work:astro dan$ bin/astro version
astro version 1.2.3 (54691ae) built 2018-12-28T14:39:18+01:00
dan-mbp-work:astro dan$ 
```

That is addressed in #14 and will be retested after this change.